graphs/bellman_ford.py

```diff
@@ -1,56 +1,40 @@
-from typing import Dict, List
-
-
-def printDist(dist, V):
-    print("Vertex Distance")
-    distances = ("INF" if d == float("inf") else d for d in dist)
-    print("\t".join(f"{i}\t{d}" for i, d in enumerate(distances)))
-
-
-def BellmanFord(graph: List[Dict[str, int]], V: int, E: int, src: int) -> int:
-    """
-    Returns shortest paths from a vertex src to all 
-    other vertices.
-    """
-    mdist = [float("inf") for i in range(V)]
-    mdist[src] = 0.0
-
-    for i in range(V - 1):
-        for j in range(E):
-            u = graph[j]["src"]
-            v = graph[j]["dst"]
-            w = graph[j]["weight"]
-
-            if mdist[u] != float("inf") and mdist[u] + w < mdist[v]:
-                mdist[v] = mdist[u] + w
-    for j in range(E):
-        u = graph[j]["src"]
-        v = graph[j]["dst"]
-        w = graph[j]["weight"]
-
-        if mdist[u] != float("inf") and mdist[u] + w < mdist[v]:
-            print("Negative cycle found. Solution not possible.")
-            return
-
-    printDist(mdist, V)
-    return src
-
-
-if __name__ == "__main__":
```

Comment:
The original uses `if __name__ == "__main__":` to avoid undesirable execution of code by pytest as discussed in CONTRIBUTING.md.